setup.py

```diff
@@ -8,22 +8,14 @@
 readme = codecs.open(os.path.join(here, 'README.rst'), encoding='utf-8').read()
 
 install_requires = [
-    'acme>=0.20,<0.21',
+    'acme>=0.21,<0.22',
     'cryptography',
+    # formerly known as acme.jose:
+    'josepy>=1.0.0',
     'pyOpenSSL',
     'pytz',
 ]
 
-if sys.version_info < (2, 7):
-    install_requires.extend([
-        'argparse',
-        'mock<1.1.0',
-    ])
-else:
-    install_requires.extend([
-        'mock',
```

Comment:
This should probably be moved to install_requires (unconditionally). I'm willing to bet the reason it's not breaking the tests is because one of our other dependencies requires it, but I'd rather not rely on that.

Reply:
Done.